test/bmc/integration_test.rb

```diff
@@ -7,6 +7,10 @@
 class BmcApiFeaturesTest < Test::Unit::TestCase
   include Rack::Test::Methods
 
+  def setup
+    Proxy::LogBuffer::Buffer.instance.send(:clear)
+  end
```

Comment:
Sure, can you introduce SmartProxyTestCase instead and do it there, then change parent class of all tests.